src/api/app/helpers/webui/request_helper.rb

```diff
@@ -7,13 +7,39 @@ module Webui::RequestHelper
     'superseded' => 'red'
   }.freeze
 
+  STATE_BOOTSTRAP_COLORS = {
+    'new' => 'info',
+    'review' => 'warning',
+    'accepted' => 'success',
+    'declined' => 'danger',
+    'revoked' => 'dark',
+    'superseded' => 'secondary'
```

Comment:
The secondary color might change over time. Let's set the actual color instead.

And then I would move this to a CSS file. Similar to what we did for the [build result colors](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/stylesheets/webui2/build-results.scss#L20).